### PR TITLE
Release 0.9.2

### PR DESCRIPTION
**Pull request by spenes**
Snyk was reporting security vulnerabilities in the Jackson libraries that are brough transitively from other libraries.
In order to get rid of these vulnerabilities, we've added those libraries as dependency to project explicitly and bump their versions.

### CODE REVIEW

**Comment by snowplowcla** [comment]
Thanks for your pull request. Is this your first contribution to a Snowplow open source project? Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).

:memo: Please visit https://docs.snowplowanalytics.com/docs/contributing/contributor-license-agreement/ to learn more and sign.

Once you've signed, please reply here (e.g. I signed it!) and we'll verify. Thanks.

File: build.sbt [hunk 1]
```diff
@@ -67,6 +67,8 @@ lazy val transformer = project.in(file("transformer"))
 
 lazy val commonDependencies = Seq(
   Dependencies.jacksonCbor,
+  Dependencies.jacksonDatabind,
+  Dependencies.jacksonScala,
```
Comment (benjben) [inline build.sbt:71]: What is the effect of adding `jackson-module-scala` to the deps? Given that jackson is a Java lib

**Review by benjben — APPROVED** [review]
:+1:

File: build.sbt (hunk 1 above)
Comment (spenes) [reply build.sbt:71]: According to [this](https://github.com/FasterXML/jackson-module-scala), it provides serializers and deserializers for Scala entities like case class, Map, Option etc. 

It was brought implicitly previously. When I've bumped jackson databind library version to `2.12.7`, it threw error due to incompatibility with jackson scala library. Therefore, I've included it too explicitly and bumped its version.

**Pull request merged**